content/services/design-systems.md

```diff
@@ -3,41 +3,179 @@ title: Integrated Design Systems
 sub: Want to reduce technical debt and scale your applications?
 index: Design Systems
 image:
-  src: pages/stacy-design-systems-training.jpg
-  alt: Stacy teaching developers at an event
+  src: pages/design-system-hero.jpg
+  alt: font sizes and previews for OddBird.net
+action:
+  text: Talk with us »
+  url: /contact/
 summary: |
-  **OddBird provides design-system training and implementation**
-  -- including strategy & design audits, automated style guides & pattern
-  libraries, component libraries, integrated toolkits, and robust
-  front-end architecture. If you're a project manager, scrum master, or
-  team leader, why not make your job easier?
+  **We partner with clients
+  to build design systems**
+  that improve team communication,
+  speed up development,
+  and lay the foundation
+  for a scalable codebase.
+  An integrated design system
+  can help your team
+  maintain consistent designs
+  across digital products
+  over the long-term --
+  finding bugs faster,
+  and weaving system updates
+  into the daily workflow.
 ---
 
 {% import 'quotes.macros.njk' as quotes %}
 {% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
 
 {{ quotes.find(
   collections.all,
-  slugs=['best', 'systems']
+  slugs=['best', 'systems'],
+  url='/work/second-measure/'
 ) }}
 
-## Integrating Tools & Processes
 
-Whether you want short-term training for your team, or active consulting
-on the design and development of your system, we have you covered. We're
-experts at building systems on a budget, and integrating tools & daily
-process to keep your system meaningful and up-to-date.
+## Packages
 
-Your team will learn to…
+We provide two integrated design system development packages.
+Purchase just the Goals & Audit package and build your system in house,
```

Comment:
@SondraE the word "just" often gives me a weird feeling. Do you think this sentence would be ok without it?

Reply:
Sure!